python/nvtx/nvtx.py

```diff
@@ -149,3 +153,46 @@ def pop_range(domain=None):
         domain is "NVTX".
     """
     libnvtx_pop_range(Domain(domain).handle)
+
+
+def start_range(message=None, color="blue", domain=None):
+    """
+    Mark the beginning of a code range.
+
+    Parameters
+    ----------
+    message : str, optional
+        A message associated with the annotated code range.
+    color : str, color, optional
+        A color associated with the annotated code range.
+        Supports
+    domain : str, optional
+        Name of a domain under which the code range is scoped.
+        The default domain name is "NVTX".
+
+    Examples
+    --------
+    >>> import time
+    >>> import nvtx
+    >>> marker_id = nvtx.start_range("my_code_range", domain="my_domain")
+    >>> time.sleep(1)
+    >>> nvtx.end_range(marker_id, domain="my_domain")
+    """
+    return libnvtx_start_range(
```

Comment:
Rather than returning an `int` here, should we return the opaque object of type `RangeId` instead? Similarly for the `end_range` function below.